src/controllers/todo-lists-controller.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const ListsService = require("../services/todo-lists-service");
+const router = express.Router()
+const listsService = new ListsService()
+const {response} = require('express');
+
+/**
+ * POST /lists/:id
+ * { "action": { "type": "DELETE", "element": 2 } }
+ *
+ * Performs any action in a list if possible. Returns the modified list or an error message.
+ */
+router.post('/:id', (req, res) => {
+    const listId = req.params.id
+    const action = req.body.action
+    const result = listsService.performAction(listId, action)
+
+    if (result.isOk)
+        return res.status.json({ list: result.list })
```

Comment:
```suggestion
        return res.json({ list: result.list })
```
El status no hace falta aca no? La respuesta default seria el 200

Reply:
uh, sí, colgué en dejarlo así porque iba a devolver otra cosa antes.